geodesy/setup.py

```diff
@@ -1,8 +1,15 @@
 #!/usr/bin/env python
 
-from distutils.core import setup
 from catkin_pkg.python_setup import generate_distutils_setup
 
+try:
+    from distutils.core import setup
+except ImportError:
+    try:
+      from setuptools import setup
+    except ImportError:
+      assert False, 'Must have distutils or setuptools installed!'
```

Comment:
No need for this logic. The only change needed in this file is to replace `from distutils.core import setup` with `from setuptools import setup` on line 3.

Reply:
Got it, will do.